.github/workflows/fossa.yaml

```diff
@@ -0,0 +1,28 @@
+name: fossa
+on:
+  repository_dispatch:
+    types: [fossa]
+  push:
+    branches:
+      - master
+      - release-*
+      - feature/*
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
+      - release-*
+      - feature/*
+  workflow_dispatch: {}
+jobs:
+  fossa-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v3
+
+      - name: "Run FOSSA Scan"
+        uses: fossas/fossa-action@main # Use a specific version if locking is preferred
+        with:
+          api-key: b88e1f4287c3108c8751bf106fb46db6 # This is a PUSH ONLY token that is safe to be shared with the public
```

Comment:
Should FOSSA test also be run in the workflow? In all other repos that is the case.
cc @shubham1172

Reply:
This is included in the workflow only now if I am missing something here.

Reply:
@amulyavarote there has to be two different steps for scan and test. Example: https://github.com/dapr/components-contrib/pull/1568